src/types/notifications/notifications.ts

```diff
@@ -4,3 +4,5 @@ export interface Notification {
   dateCreated: Date;
   isRead: boolean;
 }
+
+("");
```

Comment:
If i am not mistaken this line doesn't serve any purpose, if so remove it.

Reply:
yes it doesn't, it was there to add a change to the file for review